src/components/ContributionAssistantLabelList.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
nice ! and I'll do the same in the main price workflow
- re-add a divider at the end to separate the form and the actions
- move the "Trash" button in the footer instead of the top-left

Reply:
done here : https://github.com/openfoodfacts/open-prices-frontend/pull/1129/commits/5e9264f4b38335857490707dd517d884b4adf9aa

Reply:
Also wondering if we should create a Label component instead of a LabelList. But it can wait :)